graphene_federation/directives/shareable.py

```diff
@@ -0,0 +1,27 @@
+from typing import Callable
+
+from graphene_directives import directive_decorator
+
+from .utils import is_non_field
+from ..apollo_versions import FederationVersion, LATEST_VERSION, get_directive_from_name
```

Comment:
Use of .. Multi level relative imports, should be discouraged in favour of absolute imports